src/firebolt/async_db/_types.py

```diff
@@ -177,7 +178,7 @@ def parse_value(
     """Provided raw value and python type, parses first into python value."""
     if value is None:
         return None
-    if ctype in (int, str, float):
+    if ctype in (int, str, float, Decimal):
```

Comment:
I think that this condition is redundant.
We won't ever have ctype `Decimal`

Reply:
Done.